### PR TITLE
Add script to get another user's info

### PR DESCRIPTION
**Pull request by hawx**
Connects #1

### CODE REVIEW

File: src/scripts/tell_me_about_username.script.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      return;
+    }
+
+    robot.client.getUser(user.id)
+      .then(res => {
+        if (!res.ok && res.statusCode === 404) {
+          return response.reply(`"${user.name}" is not a user I recognise!`);
+        }
+
+        if (!('name' in res.user.team)) {
```
Comment (Codesleuth) [inline src/scripts/tell_me_about_username.script.ts:22]: The API won't send back `team` if the user is not in a team. e.g.:

``` json
HTTP/1.1 /users/U01234567

{
  "links": {
    "self": "/users/U01234567"
  },
  "data": {
    "type": "users",
    "id": "U01234567",
    "attributes": {
      "name": "hawx"
    }
  }
}
```

At the moment you'll get `TypeError: Cannot use 'in' operator to search for 'name' in undefined`

**Review by Codesleuth — CHANGES_REQUESTED** [review]

File: src/scripts/tell_me_about_username.script.ts (hunk 1 above)
Comment (hawx) [reply src/scripts/tell_me_about_username.script.ts:22]: So [this](https://github.com/TechNottingham/hubot-hackbot/blob/master/src/scripts/tell_me_about_my_team.script.ts#L10) is also broken? [And](https://github.com/TechNottingham/hubot-hackbot/blob/5a29c0ef6939895244649011f55e123667bfad6d/src/scripts/our_motto_is.script.ts#L12).

Comment (Codesleuth) [reply src/scripts/tell_me_about_username.script.ts:22]: Probably. Need to try it.

Comment (Codesleuth) [reply src/scripts/tell_me_about_username.script.ts:22]: Let's investigate that later. Will approve for now.

**Review by Codesleuth — APPROVED** [review]

**Pull request merged**